alembic/versions/76e1e55bc5c1_add_contributors.py

```diff
@@ -0,0 +1,64 @@
+"""Add contributors
+
+Revision ID: 76e1e55bc5c1
+Revises: b3767156e04a
+Create Date: 2024-04-29 06:17:03.265438
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '76e1e55bc5c1'
+down_revision = 'b3767156e04a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('contributors',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('orcid_id', sa.String(), nullable=False),
+    sa.Column('given_name', sa.String(), nullable=True),
+    sa.Column('family_name', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_index(op.f('ix_contributors_orcid_id'), 'contributors', ['orcid_id'], unique=False)
+    op.create_table('experiment_set_contributors',
+    sa.Column('experiment_set_id', sa.Integer(), nullable=False),
+    sa.Column('contributor_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['contributor_id'], ['contributors.id'], ),
+    sa.ForeignKeyConstraint(['experiment_set_id'], ['experiment_sets.id'], ),
+    sa.PrimaryKeyConstraint('experiment_set_id', 'contributor_id')
+    )
+    op.create_table('experiment_contributors',
+    sa.Column('experiment_id', sa.Integer(), nullable=False),
+    sa.Column('contributor_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['contributor_id'], ['contributors.id'], ),
+    sa.ForeignKeyConstraint(['experiment_id'], ['experiments.id'], ),
+    sa.PrimaryKeyConstraint('experiment_id', 'contributor_id')
+    )
+    op.create_table('scoreset_contributors',
+    sa.Column('scoreset_id', sa.Integer(), nullable=False),
+    sa.Column('contributor_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['contributor_id'], ['contributors.id'], ),
+    sa.ForeignKeyConstraint(['scoreset_id'], ['scoresets.id'], ),
+    sa.PrimaryKeyConstraint('scoreset_id', 'contributor_id')
+    )
+    op.drop_index('ix_mapped_variants_js', table_name='mapped_variants')
+    op.drop_index('ix_variants_js', table_name='variants')
```

Comment:
Yeah, sorry -- I'd issued the PR for a version of the code that wasn't rebased.

Reply:
I could have sworn we removed them manually, having added them manually as a hot fix. If they're still present in prod, we should drop them since they're redundant.

This isn't in the rebased version, so I'm not transferring the comment.